src/data/create_segments.py

```diff
@@ -273,12 +273,22 @@ def add_point_based_features(non_inters, inters, jsonfile,
         near = feature.near_id
         feat_type = feature.properties['feature']
 
+        if 'feat_agg' in feature.properties and 'value' in feature.properties:
+            feat_agg_type = feature.properties['feat_agg']
+        else:
+            feat_agg_type = 'default'
+
         if near:
-            if str(near) not in matches:
```

Comment:
You can pull `
if str(near) not in matches:
    matches[str(near)] = {}
` outside of the if/else, to avoid duplicating code